messaging/methods/messages/createMessage.md

```diff
@@ -31,6 +31,7 @@ When sending a group message to an invalid phone number, you may receive extrane
 | applicationId | Yes       | `string`                                                       | The ID of the Application your `from` number is associated with in the Bandwidth Phone Number Dashboard.                                                                                                                                                                                                                                                                                                                      |
 | media*        | No (required if no text is provided)       | `array`                                                        | A list of URLs to include as media attachments as part of the message. If this field is included, the message will be sent as MMS no matter the number of recipients.<br>Refer to the [What MMS message size limits must be adhered to?](https://support.bandwidth.com/hc/en-us/articles/360014235473-What-MMS-message-size-limits-must-be-adhered-to-) article to learn more about file size limitations and best practices. |
 | tag           | No        | `string`                                                       | Any string which will be included in the callback events of the message. (max 1024 Chars)                                                                                                                                                                                                                                                                                                                                     |
+| priority      | No        | `string` | The message's priority, currently for toll-free or short code SMS only. Accepted values are `"default"` or `"high"`. Messages with a priority value of `"high"` are given preference over your other traffic. |
```

Comment:
Should we say if no value is specified in the POST request body that it will default to `default` - i know thats overkill on redundancy but i wouldnt put it past getting asked "what happens if i dont put priority - will bandwidth just assume i want all of my messages to be high priority?"